src/Middleware/CORS/src/Infrastructure/CorsMiddleware.cs

```diff
@@ -129,7 +129,7 @@ public Task Invoke(HttpContext context, ICorsPolicyProvider corsPolicyProvider)
             return InvokeCore(context, corsPolicyProvider);
         }
 
-        private async Task InvokeCore(HttpContext context, ICorsPolicyProvider corsPolicyProvider)
```

Comment:
This can be merged into the `Invoke` method. They were previously separated to avoid the state machine, but now there's no real need to.